go/interpreter/lfvm/interpreter_test.go

```diff
@@ -652,16 +652,12 @@ func TestStepsProperlyHandlesJUMP_TO(t *testing.T) {
 
 func TestStepsDetectsNonExecutableCode(t *testing.T) {
 	// Create execution context.
-	instructions := []struct {
-		instruction []Instruction
-		status      status
-	}{
-		{[]Instruction{{NUM_EXECUTABLE_OPCODES - 1, 0x0101}, {DATA, 0x0001}, {STOP, 0}}, statusStopped},
-		{[]Instruction{{NUM_EXECUTABLE_OPCODES, 0}}, statusError},
-		{[]Instruction{{NUM_EXECUTABLE_OPCODES + 1, 0}}, statusError},
+	opCodes := []OpCode{
+		NUM_EXECUTABLE_OPCODES,
+		NUM_EXECUTABLE_OPCODES + 1,
```

Comment:
I think it is important to have data input that does not trigger the property.

By not testing an executable opcode, we cannot test whenever the function is just returning always true. 
nit: I would add INVALID, NOP, and DATA because they are located inside of the "range"

Reply:
You are right, there should always be tests for the positive and negative cases. And this should be improved here. However, I think all the unit tests in this file are a mess and they should get a complete do-over -- and I plan to do so before the release. Thus, I do not really care about these changes now.